docs/content/preview/yugabyte-platform/security/ldap-authentication-platform.md

```diff
@@ -97,7 +97,7 @@ Consider the following example:
     # numEntries: 1
     ```
 
-    If instead you see a message similar to `ldap_sasl_bind(SIMPLE): Can't contact LDAP server (-1)`, then you have a network, certificate, or binding (authentication) problem.
+    If instead you see a message similar to `ldap_sasl_bind(SIMPLE): Can't contact LDAP server (-1)`, then you have a network, certificate, or binding (authentication) problem.  See [Troubleshooting](#troubleshooting) below.
```

Comment:
Page-scope positional indicators cannot be used in technical documentation. So, here it should be:

See [Troubleshooting](#troubleshooting).

Reply:
I think github clobbered your comment... do you mean `#troubleshooting` ?

Reply:
What I meant, there shouldn't be "below". Just 
See Troubleshooting.